scene/gui/option_button.cpp

```diff
@@ -230,6 +230,12 @@ void OptionButton::set_item_icon(int p_idx, const Ref<Texture2D> &p_icon) {
 	_queue_update_size_cache();
 }
 
+void OptionButton::set_item_icon_max_width(int p_idx, int p_width) {
+	popup->set_item_icon_max_width(p_idx, p_width);
+
+	_queue_update_size_cache();
```

Comment:
Also the implementation is wrong. The max width does not even affect the OptionButton's size.

Reply:
`set_expand_icon()` handles the button's size.

Reply:
okay, yeup, I see. Removing.